### DISCUSSION TITLE
Arduino mega for V3 gone from stock, what can I do?

### DISCUSSION THREAD

**Discussion by Matt90116** [sanni/cartreader#333] [Need help]
Hi the Arduino mega used for the v3 revision build is not available from Ali express right now. Im not sure if these devices do go out of stock and return? If I cant buy one I dont know how I am going to build the reader. Any suggestions? Can I use another Arduino board? I have seen others of similar fashion from arduino, but they dont match 100% I know your not thw manufactuer but, maybe these will come back in stock if there is there is no replacement?

**Comment by Dakkaron** [comment]
Have a look at AliExpress: https://m.aliexpress.com/wholesale/arduino+mega.html?osf=auto_suggest

**Comment by InvaderZimm** [comment]
can you use a elegoo arduino instead if its what you have and which components  would you remove from it because it looks a little different from the one in the build 
![20220311_004522](https://user-images.githubusercontent.com/77585654/157833293-f5c0157c-a860-4aae-96ac-aff8fedf1f22.jpg)

**Comment by sanni** [comment]
Yes you can use this one. Desolder these parts:
![image](https://dl.dropboxusercontent.com/s/38ad01xbej3k1rx/0.png?dl=1)   
https://github.com/sanni/cartreader/wiki/How-to-build-the-PCB

**Comment by InvaderZimm** [comment]
![20220312_021407](https://user-images.githubusercontent.com/77585654/158013996-2b163b95-7292-437a-826d-3e62166dbc8e.jpg)
ok so got those off and one wire on do i solder the second one to this spot here top or bottom and thank you for helping with the first question

**Comment by sanni** [comment]
Bottom:
![image](https://dl.dropboxusercontent.com/s/7p3ienmtzifvnny/9.png?dl=1)

**Reply by InvaderZimm** [reply]
thank you again for the help
